x-pack/platform/plugins/shared/lens/public/runtime_state/converters/raw_color_mappings/converter.ts

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ColorMapping } from '@kbn/coloring';
+import { MultiFieldKey, RangeKey, SerializedValue } from '@kbn/data-plugin/common';
+import { DeprecatedColorMappingConfig } from './types';
+import { GenericIndexPatternColumn } from '../../../async_services';
+
+/**
+ * Converts old stringified colorMapping configs to new raw value configs
+ */
+export function convertToRawColorMappings(
```

Comment:
Only this and the `isDeprecatedColorMapping` function are exported from the index.ts if not used externally you can probably remove the export

Reply:
True, my plan was to test the other functions in isolation but I could just do that with those 2 exports.

Reply:
I'm fine whatever you decide, it was just a nit pick

Reply:
Removed exports in 85c970f3bdbcf2820a7e7ec23e01f773a3d814e6